Dockerfile

```diff
@@ -11,28 +11,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-# For details and docs - see https://github.com/phusion/baseimage-docker#getting_started
-
-FROM ubuntu:14.04
+FROM alpine
+LABEL maintainer "Neil Jerram <neil@tigera.io>"
 
 CMD ["/sbin/my_init"]
 
 ENV HOME /root
 
-# Uncomment these lines and comment the section underneath to allow faster
-# rebuilds when making changes to the scripts.
-# The early scripts take a long time to run but change infrequently so
-# putting them on a their own lines allow developers to take advantage of
-# Docker's layer caching. The downside is much larger images.
-#ADD /image/buildconfig /build/buildconfig
-#ADD /image/my_init /build/my_init
-#ADD /image/install.sh /build/install.sh
-#RUN /build/install.sh
-#ADD /image/system_services.sh /build/system_services.sh
-#RUN	/build/system_services.sh
-#ADD /image/cleanup.sh /build/cleanup.sh
-#RUN	/build/cleanup.sh
+RUN echo "ipv6" >> /etc/modules
+RUN echo "http://dl-1.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories; \
```

Comment:
As discussed offline, if we're going to use edge, I think we should use an edge base image, which should remove the need to add these repos.